lib/Validator.php

```diff
@@ -7,7 +7,7 @@
 final class Validator implements ValidatorInterface
 {
     const XSD_FATTURA_ORDINARIA_1_2         = 'Schema_VFPR12.xsd';
-    const XSD_FATTURA_ORDINARIA_LATEST      = 'Schema_VFPR12.xsd';
+    const XSD_FATTURA_ORDINARIA_LATEST      = 'Schema_VFPR1.2.1a.xsd';
```

Comment:
```suggestion
    const XSD_FATTURA_ORDINARIA_1_2_1       = 'Schema_VFPR121a.xsd';
    const XSD_FATTURA_ORDINARIA_LATEST      = 'Schema_VFPR121a.xsd';
```
Ciao, puoi aggiungere come indicato anche una costante dedicata alla 1.2.1a per favore?
Inoltre ti chiedo di mantenere lo schema per la versione come la precedente, e quindi di rinominare il file